File: Angular/src/app/models/purchase.ts
export class Purchase {
    purchaseId:number;
    customerId:number;
    purchaseDate:Date;
    amountPay:number;
    remark:string;
    constructor(purchaseId:number,
        customerId:number,
        purchaseDate:Date,
        amountPay:number,
        remark:string){
            this.amountPay=amountPay;
            this.customerId=customerId;
            this.purchaseDate=purchaseDate;
            this.purchaseId=purchaseId;
            this.remark=remark;

    }
}

File: C#/Dal/Models/Purchase.cs
using System;
using System.Collections.Generic;

namespace Dal_Repository.Models;

public partial class Purchase
{
    public int PurchaseId { get; set; }

    public int CustomerId { get; set; }

    public DateTime PurchaseDate { get; set; }

    public decimal AmountPay { get; set; }

    public string? Remark { get; set; }

    public virtual Customer Customer { get; set; } = null!;

    public virtual ICollection<PurchaseDetail> PurchaseDetails { get; set; } = new List<PurchaseDetail>();
}
